docs/static/upgrading.asciidoc

```diff
@@ -77,3 +77,20 @@ of workers by passing a command line flag such as:
 
 [source,shell]
 bin/logstash `-w 1`
+
+=== Upgrading Logstash to 2.2
+
+Logstash 2.2 re-architected the pipeline stages to provide more performance and help future enhancements in resiliency. 
+The new pipeline introduced micro-batches, processing groups of events at a time. The default batch size is 
+125 per worker. Also, the filter and output stages are executed in the same thread, but still, as different stages.
```

Comment:
Also add 'this will by default also be the bulk size used for the elasticsearch output. The Elasticsearch output's `flush_size` now acts only as a maximum bulk size (still defaulting to 500).'